### ISSUE TITLE
(aws-stepfunctions-tasks): Add support for SQS ReceiveMessage and DeleteMessage tasks

### ISSUE THREAD

**Issue by Gtofig** [aws/aws-cdk#23630]
### Describe the feature

Step Functions support `arn:aws:states:::aws-sdk:sqs:receiveMessage` and `arn:aws:states:::aws-sdk:sqs:deleteMessage` resource types now. Please add CDK support for it.

### Use Case

I would like to be able to use L2 CDK constructs for all tasks, and ReceiveMessage and DeleteMessage APIs for SQS are currently missing

### Proposed Solution

Create new `SqsReceiveMessage` and `SqsDeleteMessage` classes (similar to `SqsSendMessage`), that will implement `arn:aws:states:::aws-sdk:sqs:receiveMessage` and `arn:aws:states:::aws-sdk:sqs:deleteMessage` resources respectively.

Currently the workaround is to use `CustomState` class

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### CDK version used

2.59.0

### Environment details (OS name and version, etc.)

N/A

**Comment by pahud** [comment]
Agree. Any PR submission for that would be highly appreciated and before we have that support, I think the [aws sdk support](https://github.com/aws/aws-cdk/tree/main/packages/%40aws-cdk/aws-stepfunctions-tasks#aws-sdk) should be a workaround.
